### DISCUSSION TITLE
Level of support for JSON Schema  and  OpenAPI v3 specifications

### DISCUSSION THREAD

**Discussion by lfvjimisola** [koxudaxi/datamodel-code-generator#1852] [Q&A]
Hi,

I might have missed it in the documentation but does datamodel-codegen support the full specifications for JSON Schema and OpenAPI v3? If not, is there a list of what it is that is (currently) not supported?

BR,
Jimisola

**Comment by koxudaxi** [comment]
Not yet. We implement the spec as possible as. We should provide the documents in the future.
